src/main/resources/i18n/messages.properties

```diff
@@ -98,24 +98,19 @@ ranked1v1.rankingFormat=#{0,number,#}
 ranked1v1.notification.title=1v1 ranked match available
 ranked1v1.notification.message=Click here to open the 1v1 tab
 teammatchmaking.leaveParty=Leave party
-teammatchmaking.title=Team matchmaking
 teammatchmaking.invitePlayer=Invite player
 teammatchmaking.invite=Invite
-teammatchmaking.availableQueues=Available queues
-teammatchmaking.playersInQueue=In Queue: {0,number,#}
-teammatchmaking.teamSize=Team size: {0,number,#}
-teammatchmaking.joinQueue=Join
-teammatchmaking.leaveQueue=Leave
+teammatchmaking.playersInQueue=IN QUEUE: {0,number,#}
 teammatchmaking.queuePopTimer={0,number,0}:{1,number,00}
-teammatchmaking.chat=Chat / Map pool?
-teammatchmaking.statistics=Statistics / Chat?
-teammatchmaking.rating=Placement (1v1)
-teammatchmaking.gameCount=Games: {0,number,#}
+teammatchmaking.rating=RATING: {0,number,#}
+teammatchmaking.1v1Rating=1V1 RATING: {0,number,#}
+teammatchmaking.teamRating=TEAMGAME RATING: {0,number,#}
+teammatchmaking.gameCount=GAMES PLAYED: {0,number,#}
 teammatchmaking.notification.invite.title=Invited to party
 teammatchmaking.notification.invite.message={0} has invited you to join their party
 teammatchmaking.notification.invite.join=Join
-teammatchmaking.ready=You are ready!
-teammatchmaking.notReady=Ready up!
+teammatchmaking.ready=SELECT QUEUES BELOW AND WAIT FOR MATCH
+teammatchmaking.notReady=PLAY FORGED ALLIANCE FOREVER
```

Comment:
I'd remove the Forged alliance forever, this isn't the only way to play

Reply:
Having the single short word "play" on this button looks stupid. But I am open for alternatives.
And it is the primary intended way, though.

Reply:
I don't know why we would write Play here anyways. The user is in the play tab and click on matchmaking. And they are obviously running the Forged Alliance Forever client.
Usually a button is labled as what it does. And in this case, it joins a queue.